README.md

```diff
@@ -15,67 +17,21 @@ For each of the three exchanges `USD`, `EUR` and `JPY`, this demo makes use of t
 - `CrytoTraderQuotes{USD/EUR/JPY}`
 - `CryptoTraderQuotesAvg{USD/EUR/JPY}`
 
-The below steps will describe on how to deploy the node and UI application.
+The below steps will describe on how set up the frontend.
 
-# 2. Prerequisites
+# 1. Prerequisites
 
-Clone tutorial-cryptotrading.
+Clone or Fork this repo.
 
 `nodejs` and `npm` must be installed on your system.
 
-`trades` collection should be present.
-
-# 3. How to run app(UI) locally
+# 2. How to run app(UI) locally
```

Comment:
```suggestion
# 2. Run the App Locally
```

Reply:
Why are these all h1s? Is that a Git markdown thing? Because usually the title is h1 and everything after it is h2 or below.

Also, in headings, "how to" is almost always unnecessary.